splinepy/helpme/create.py

```diff
@@ -295,10 +304,47 @@ def parametric_view(spline, axes=True):
         parametric_bounds=p_bounds, physical_bounds=p_bounds
     )
 
-    # loop through knot vectors and insert missing knots
-    for i, kv in enumerate(spline.unique_knots):
-        if len(kv) > 2:
-            para_spline.insert_knots(i, kv[1:-1])
+    # process to create conforming para_view splines
+    if conform:
+        # conforming spline - rational?
+        if spline.is_rational:
+            para_spline = para_spline.nurbs
+
+        # conforming spline - bezier?
+        if not spline.has_knot_vectors:
+            dict_spline = para_spline.todict()
+            dict_spline.pop("knot_vectors")
+            para_spline = type(spline)(**dict_spline)
+
+        # conforming degrees
+        d_diff = spline.degrees - para_spline.degrees
+        if any(d_diff):
+            # turn difference in degrees into degree query
+            d_query = []
+            for i, dq in enumerate(d_diff):
+                d_query += [i] * dq
+            para_spline.elevate_degrees(d_query)
+
+        # process knots to insert
+        if spline.has_knot_vectors:
+            for i, (kv, d) in enumerate(
+                zip(spline.knot_vectors, spline.degrees)
+            ):
+                n_repeating = int(d + 1)
+                query = kv[n_repeating:-n_repeating]
+                if len(query) > 0:
+                    para_spline.insert_knots(i, query)
```

Comment:
This can be simplified, as starting #214, you can just pass all knots at the same time to simplify the syntax.

Reply:
I don't see any related changed in #214 but you can help me for next pr